src/client/tile.cpp

```diff
@@ -1811,6 +1811,23 @@ bool TextureSource::generateImagePart(std::string part_of_name,
 	return true;
 }
 
+/*
+	Calculate the color of a single pixel drawn on top of another pixel.
+
+	This is a little more complicated than just video::SColor::getInterpolated
+	because getInterpolated does not handle alpha correctly.  For example, a
+	pixel with alpha=64 drawn atop a pixel with alpha=128 should yield a
+	pixel with alpha=160, while getInterpolated would yield alpha=96.
+*/
+static video::SColor blit_pixel(const video::SColor &src_c, const video::SColor &dst_c, u32 ratio)
```

Comment:
Minetest (largely) follows the `camelCase` naming convention for functions. It'd be nice if this could is renamed to smth. like `blitPixel` instead.

Reply:
I'm aware that the other related functions in this file use the underscore convention, but I think it's better if we don't continue this style and use the correct naming convention (`camelCase`, in this case) instead.